### DISCUSSION TITLE
podman-install on v4.0.6 - missing required GLIBC

### DISCUSSION THREAD

**Discussion by zenlord** [unifi-utilities/unifi-common#623] [Q&A]
Hi,
Trying to run podman on my UDM Pro, something that would be completely impossible without these utilities, so thank you very much!
I have successfully installed the udm-boot service and after adapting the `/on_boot.d/00-podman.sh` script to reflect the changes in the $DESIRED_ZIPFILE structure (it no longer contains a podman-install.zip, but just another directory), the script still fails because podman is missing a required GLIBC_VERSION:
```
# podman version
podman: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by podman)
podman: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by podman)
podman: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by podman)
```

Do you have another trick up your sleeve to get around this one? :). If you are interested, I can submit a PR to correct the 00-podman.sh script.

**Comment by m4n63** [comment]
Podman has not worked for ages, you need to use this method:
https://github.com/unifi-utilities/unifios-utilities/tree/main/nspawn-container
